app/controllers/application_controller.rb

```diff
@@ -22,22 +22,31 @@ def admin_path?
   def opengraph
     return if admin_path?
 
-    @opengraph ||= OpengraphBuilder.new({
-                                          'og': {
-                                            'site_name': t('meta.site.name'),
-                                            'title': t('meta.site.title'),
-                                            'description': t('meta.site.description'),
-                                            'url': request.url,
-                                            'type': 'website',
-                                            'image': URI.join(root_url, helpers.image_path(t('meta.image'))),
-                                            'image:alt': t('meta.image_alt'),
-                                            'locale': I18n.locale
-                                          },
-                                          'twitter': {
-                                            'site': t('meta.twitter.site'),
-                                            'creator': t('meta.twitter.creator')
-                                          }
-                                        })
+    @opengraph ||= OpengraphBuilder.new('og': og_tags, 'twitter': twitter_tags)
+  end
+
+  def og_tags
+    {
+      'site_name': t('meta.site.name'),
+      'title': t('meta.site.title'),
+      'description': t('meta.site.description'),
+      'url': request.url,
+      'type': 'website',
+      'image': URI.join(root_url, helpers.image_path(t('meta.image'))),
+      'image:alt': t('meta.image_alt'),
+      'image:height': t('meta.image_height'),
+      'image:width': t('meta.image_width'),
+      'locale': 'en_GB',
+      'locale:alternate': "#{I18n.locale}_#{I18n.locale.upcase}"
```

Comment:
Why are we using an hardcoded string for locale while using I18n for locale:alternate?

Reply:
sorry, this line was later removed